source/dofs/dof_renumbering.inst.in

```diff
@@ -288,11 +288,6 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
       template void
       random<DoFHandler<deal_II_dimension>>(DoFHandler<deal_II_dimension> &);
 
-      template void
-      compute_random<DoFHandler<deal_II_dimension>>(
-        std::vector<types::global_dof_index> &,
-        const DoFHandler<deal_II_dimension> &);
```

Comment:
Don't you have to revert the changes to file as well?

Reply:
Of course! :pensive: